dtfs-central-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -9,6 +9,10 @@ const mongoIdValidation = param('_id').isMongoId().withMessage("Invalid MongoDB
 
 exports.mongoIdValidation = [mongoIdValidation];
 
+const sqlIdValidation = param('id').isInt().withMessage("Invalid 'id' path param provided");
```

Comment:
Hello I'm dumb and haven't looked up what our IDs are, are we using an incremental ID for our utilisation reports atm?

Reply:
The SQL entity is using the `@PrimaryGeneratedColumn` decorator on the id which by my understanding defaults to auto incrementing IDs